docs/user-guide/cli-ftpplugin.md

```diff
@@ -34,15 +36,21 @@ Before you install the plug-in, meet the [Software requirements for Zowe CLI plu
 Use one of the following methods to install or update the plug-in:
 
 - [Installing plug-ins from an online registry](cli-installplugins.md#installing-plug-ins-from-an-online-registry)
-
 - [Installing plug-ins from a local package](cli-installplugins.md#installing-plug-ins-from-a-local-package)
 
 ## Creating a user profile
 
 After you install the plug-in, create an FTP profile. An FTP profile is recommended to issue commands via FTP. FTP profiles contain your host, port, user name, and password to connect to z/OS using FTP. You can create multiple profiles and switch between them as needed.
 
-**Follow these steps:**
-1.  Install the z/OS FTP Plug-in for Zowe CLI
+### Creating plug-in profiles using a configuration file
+
+When you issue various `zowe config` commands, such as `init`, `auto-init`, and `convert-profiles`, they create a `zowe.config.json` configuration file. When you install the FTP plug-in and then issue a `zowe config` command **[correct?]**, the command creates an entry for an FTP profile in your `zowe.config.json` file.
```

Comment:
@adam-wolfe - please respond

Reply:
This is true, but I would also consider my other comment on how the wording could be changed.